src/jit.c

```diff
@@ -2026,10 +2036,28 @@ void parse_branch_history_table(struct jit_state *state,
     }
 }
 
+void emit_jit_update_timer(struct jit_state *state)
```

Comment:
How about renaming this function to `emit_jit_inc_timer`? It emphasizes the increment operation, which is the core action (incrementing the timer).

Reply:
In some other implementations, it might only be updated at the end of a basic block, but we update it every time for convenience here. So I would prefer to keep it that way.